securedrop/journalist.py

```diff
@@ -9,4 +9,4 @@
 
 if __name__ == "__main__":  # pragma: no cover
     debug = getattr(config, 'env', 'prod') != 'prod'
-    app.run(debug=debug, host='0.0.0.0', port=8081)
+    app.run(debug=debug, host='127.0.0.1', port=8081)
```

Comment:
This will break dev (I think) since the container is part of the `10.0.0.0/8` network. Prod also doesn't start SD with main. It uses WSGI to start the app, so the debug check here is actually unnecessary. Also fun fact, adding that line was my first ever commit to SD.

Reply:
Just tried this out, and you are correct. Given Flask's inability to bind to multiple interfaces, and WSGI is used to start the app, I will revert the change and add the nosec annotation. Let me know if you disagree

Reply:
Nope, that's exactly what I would do.